core/arch/arm/plat-stm32mp1/conf.mk

```diff
@@ -92,6 +91,10 @@ CFG_STM32MP_PANIC_ON_TZC_PERM_VIOLATION ?= y
 
 # SiP/OEM service for non-secure world
 CFG_STM32_BSEC_SIP ?= y
+CFG_STM32MP1_SCMI_SIP ?= y
+ifeq ($(CFG_STM32MP1_SCMI_SIP),y)
+$(call force,CFG_SCMI_MSG_SMT_FASTCALL_ENTRY,y)
+endif
```

Comment:
What does this new config flag bring us?

Reply:
CFG_STM32MP1_SCMI_SIP=y/n allows to embed or not a SiP SMC entry point in STM32MP1 SCMI server.

I'm working on an alternate entry for SCMI server, that is a generic SCMI PTA interface (https://github.com/OP-TEE/optee_os/pull/4533) and at some point, i may disable the SiP SMC entry and keep only the SCMI PTA one.

Reply:
> CFG_STM32MP1_SCMI_SIP=y/n allows to embed or not a SiP SMC entry point in STM32MP1 SCMI server.

Yes but which one? It doesn't tell. `CFG_SCMI_MSG_SMT_FASTCALL_ENTRY` (and the future flag you will use for the PTA mode) does tell everything, so I think `CFG_STM32MP1_SCMI_SIP` is useless.

Reply:
Oh sorry, I see a change is missing! the one that uses CFG_STM32MP1_SCMI_SIP! i'll push a fixup.

Reply:
I've pushed the `sip_service()` part of the patch. sorry :confused: 
I've changed a bit the platform **conf.mk**. Better like this I guess.